CMSgov/onemac
# OY2 19963: Create Details Page: 1915(b) Waiver Renewal

Story: https://qmacbis.atlassian.net/browse/OY2-19963
Endpoint: See github-actions bot comment

### Details
Create the Waiver Renewal Details page for the Package View.

### Changes
- updated the routing to go to the Waiver Renewal Details page
- created the Waiver Renewal Detail page
- Updated the detail title "Submission Date" to "Initial Submission Date" for all Detail Pages
- Updated Detail Section so it does not show the title for the Formal RAI Responses unless there are Formal RAI Responses
- added a bunch of unit tests that apparently don't help with coverage :P 

### Test Plan
1. Login as any user that can see Waivers
2. Open a Waiver Renewal
3. Verify that the details and texts match the AC
